### PR TITLE
update to newest library version

### PR DESCRIPTION
**Pull request by Daniel-K-Ivanov**

### CODE REVIEW

File: backend/server.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ app.use('/', express.static('public'));
 app.post('/fiatPayment', async (request, response, next) => {
     try {
         const fiatPaymentData = await getFiatData();
-        fiatPaymentData.shopper = "5c3e090cfd77ee6054c03883";  // Hard-coded shopper ID
+        fiatPaymentData.shopper = "5c3e0757c67b1a3deadb855a";  // Hard-coded shopper ID
```
Comment (vsavovlime) [inline backend/server.js]: What you think to export shopper id to config.json?

File: backend/server.js [hunk 2]
```diff
@@ -40,7 +40,7 @@ app.post('/fiatPayment', async (request, response, next) => {
 app.post('/relayedPayment', async (request, response, next) => {
     try {
         const relayedPaymentData = await getRelayedData();
-        relayedPaymentData.shopper = "5c3e090cfd77ee6054c03883";  // Hard-coded shopper ID
+        relayedPaymentData.shopper = "5c3e0757c67b1a3deadb855a";  // Hard-coded shopper ID
```
Comment (vsavovlime) [inline backend/server.js]: Same here as previous comment.

File: backend/server.js (hunk 1 above)
Comment (vsavovlime) [inline backend/server.js]: What you think to export shopper id to config.json?

**Review by vsavovlime — CHANGES_REQUESTED** [review]

File: backend/server.js (hunk 2 above)
Comment (Daniel-K-Ivanov) [reply backend/server.js]: I've added the ID to the config.js

File: backend/server.js (hunk 1 above)
Comment (Daniel-K-Ivanov) [reply backend/server.js]: I've added the ID to the config

**Review by vsavovlime — APPROVED** [review]

**Pull request merged**